### PR TITLE
fix - update model to work with primary only dataset

### PR DESCRIPTION
**Pull request by Ragansu**
This PR has few fixes.
* Changed post process to systematics.
* fixed the bug in train test split (now uses the help of sklearn train test split)
* removed old post process code. 

RC

### CODE REVIEW

**Comment by Ragansu** [comment]
image

**Review by AvencastF — COMMENTED** [review]
All good for me. Running local test.

File: sample_code_submission/model_XGB.json
Comment (AvencastF) [inline sample_code_submission/model_XGB.json:1]: I suggest removing the model for now. Once we launch the challenge, we can add one for example.

File: sample_code_submission/model_XGB.pkl
Comment (AvencastF) [inline sample_code_submission/model_XGB.pkl:1]: The same as before. I suggest removing the model for now. Once we launch the challenge, we can add one for example.

File: sample_code_submission/saved_info_model_XGB.pkl
Comment (AvencastF) [inline sample_code_submission/saved_info_model_XGB.pkl:1]: I suggest removing the model for now. Once we launch the challenge, we can add one for example.

**Comment by Ragansu** [comment]
You have a point also some times saved models causes issues (mostly for tensorflow) when moving between windows and Unix/linux.

**Comment by AvencastF** [comment]
> You have a point also some times saved models causes issues (mostly for tensorflow) when moving between windows and Unix/linux.

That's unexpected to me. Never tried Windows before...

**Comment by AvencastF** [comment]
Sure. Sounds good to me. So we should keep these trained information or delete them for now?

**Review by AvencastF — APPROVED** [review]
All good.

**Comment by Ragansu** [comment]
I build on this PR and improved the memero management on a different PR https://github.com/FAIR-Universe/HEP-Challenge/pull/97

Hence I am clsong this PR. 

Thank you all
RC

**Pull request closed without merging**